src/track/trackmetadatataglib.cpp

```diff
@@ -838,6 +838,29 @@ void readTrackMetadataFromID3v2Tag(TrackMetadata* pTrackMetadata,
         parseTrackPeak(pTrackMetadata,
                 toQString(pTrackPeakFrame->fieldList()[1]));
     }
+    TagLib::ID3v2::FrameList ratingFrame = tag.frameListMap()["POPM"];
+    int rating = 0;
+    if(!ratingFrame.isEmpty()) {
+        // RatingString "traktor@native-instruments.de rating=255 counter=2"
+
+        QString sRating = TStringToQString(ratingFrame.front()->toString());
+        sRating = sRating.section("=",1,2).left(3);
+        float fRating = sRating.toInt();
+        rating = ceil(fRating/51);
+
+        // Calc rating
+        //
+        // NI - Rating
+        // 255 = 5 | 204 = 4 | 153 = 3 | 102 = 2 | 51 = 1 | 0 = 0
+        //
+        // Banshee - Rating
+        // 255 = 5 | 192 = 4 | 128 = 3 | 64 = 2 | 1 = 1
+        //
+        //  ==> Rating = ceil ( X / 51 )
```

Comment:
This has mapping has to be justified by more examples. If it works for 2 applications it may fail for the 3rd.

Ratings are a really complicated topic if you study the various file formats. Not taking into account the various interpretations and implementation of the standards that can be found in applications. This is the reason why I kept the Mixxx-specific rating in TrackInfoObject.